datasets/openslr/README.md

```diff
@@ -0,0 +1,316 @@
+
+#### SLR64: Crowdsourced high-quality Marathi multi-speaker speech data set
+This data set contains transcribed high-quality audio of Marathi sentences recorded by volunteers. The data set 
+consists of wave files, and a TSV file (line_index.tsv). The file line_index.tsv contains a anonymized FileID and 
+the transcription of audio in the file.
+
+The data set has been manually quality checked, but there might still be errors.
+
+Please report any issues in the following issue tracker on GitHub. https://github.com/googlei18n/language-resources/issues
+
+See LICENSE file for license information.
+
+Copyright 2018, 2019 Google, Inc.
+#### SLR65: Crowdsourced high-quality Tamil multi-speaker speech data set
+This data set contains transcribed high-quality audio of Tamil sentences recorded by volunteers. The data set 
+consists of wave files, and a TSV file (line_index.tsv). The file line_index.tsv contains a anonymized FileID and 
+the transcription of audio in the file.
+
+The data set has been manually quality checked, but there might still be errors.
+
+Please report any issues in the following issue tracker on GitHub. https://github.com/googlei18n/language-resources/issues
+
+See LICENSE file for license information.
+
+Copyright 2018, 2019 Google, Inc.
+
+#### SLR66: Crowdsourced high-quality Telugu multi-speaker speech data set
+This data set contains transcribed high-quality audio of Telugu sentences recorded by volunteers. The data set 
+consists of wave files, and a TSV file (line_index.tsv). The file line_index.tsv contains a anonymized FileID and 
+the transcription of audio in the file.
+
+The data set has been manually quality checked, but there might still be errors.
+
+Please report any issues in the following issue tracker on GitHub. https://github.com/googlei18n/language-resources/issues
+
+See LICENSE file for license information.
+
+Copyright 2018, 2019 Google, Inc.
+
+#### SLR69: Crowdsourced high-quality Catalan multi-speaker speech data set
+This data set contains transcribed high-quality audio of Catalan sentences recorded by volunteers. The data set 
+consists of wave files, and a TSV file (line_index.tsv). The file line_index.tsv contains a anonymized FileID and 
+the transcription of audio in the file.
+
+The data set has been manually quality checked, but there might still be errors.
+
+Please report any issues in the following issue tracker on GitHub. https://github.com/googlei18n/language-resources/issues
+
+See LICENSE file for license information.
+
+Copyright 2018, 2019 Google, Inc.
+
+### Supported Tasks and Leaderboards
+
+[Needs More Information]
+
+### Languages
+
+Javanese, Khmer, Nepali, Sundanese, Malayalam, Marathi, Tamil, Telugu, Catalan
+
+## Dataset Structure
+
+### Data Instances
+
+A typical data point comprises the path to the audio file, called path and its sentence. 
+
+#### SLR41, SLR42, SLR43, SLR44, SLR63, SLR64, SLR65, SLR66, SLR69  
+```
+{
+  'path': 'suf_00297_00037352660'
```

Comment:
isn't this supposed to be a path to a wav file ?

Reply:
yes, this should be a path to a wav file in .cache directory. I will update the documentation accordingly, thanks.